DTHPool.sol

```diff
@@ -222,7 +235,7 @@ contract DTHPool is DTHPoolInterface, Token, usingOraclize {
 
         var (,,,votingDeadline, ,,,,newCurator) = dao.proposals(_proposalID);
 
-        if (votingDeadline < now || newCurator) {
+        if ((votingDeadline < now) || newCurator ) {
```

Comment:
no need for this parentheses here, please remove and no need for the extra whitespace either.